chia/wallet/wallet_state_manager.py

```diff
@@ -848,23 +868,21 @@ async def is_standard_wallet_tx(self, coin_state: CoinState) -> bool:
 
     async def handle_cat(
         self,
-        curried_args: Iterator[Program],
+        cat_data: CATCoinData,
```

Comment:
The names of these parameters might be better expressed as `parent_spend_data` or something like that, you don't want to give the impression that this is the current inner_puzzle etc.